src/Streamly.hs

```diff
@@ -210,7 +210,8 @@ module Streamly {-# DEPRECATED "Please use \"Streamly.Prelude\" instead." #-}
 where
 
 import Data.Semigroup (Semigroup(..))
-import Streamly.Internal.Data.SVar (MonadAsync, Rate(..))
+import Streamly.Internal.Control.Concurrent (MonadAsync)
+import Streamly.Internal.Data.SVar (Rate(..))
```

Comment:
We'll have to maintain deprecated modules? Do we remove them after a few releases after deprecating them?

Reply:
We will remove after a while.